### ISSUE TITLE
Regression Test: aria-live styling changes

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/qa#1373]
In https://github.com/phetsims/a11y-research/issues/203, the styling around aria-live elements changed to improve the output with VoiceOver. It seems to improve issues like https://github.com/phetsims/number-pairs/issues/438, https://github.com/phetsims/molecule-polarity/issues/239, https://github.com/phetsims/calculus-grapher/issues/399.

Could you please test all supported screen readers in the following sims  to make sure that responses are still spoken correctly?

- [x] calculus-grapher VO
- [x] calculus-grapher NVDA
- [x] calculus-grapher Jaws

- [x] molecule-polarity VO
- [x] molecule-polarity NVDA
- [x] molecule-polarity Jaws

- [x] vector-addition VO
- [x] vector-addition NVDA
- [x] vector-addition Jaws

This changes how responses in the yellow box of the a11y view work. If there is a regression, it will be related to that. Are there any responses that used to be heard but are now missing? Any responses that get duplicated? Are you able to reproduce https://github.com/phetsims/calculus-grapher/issues/399?

The design in number-pairs and molecule-polarity has changed so we cannot test against those anymore.

Thank you!

**Comment by vperezcadavid99** [comment]
I confirmed this issue too for vector addition with NVDA.
Even when interacting with other elements of the simulation, deleting, or changing screens, the description continues.

https://github.com/user-attachments/assets/07ac4bc1-78f9-42a8-808d-95246292b93d
